0.10/Dockerfile

```diff
@@ -1,8 +1,9 @@
-FROM buildpack-deps
+FROM buildpack-deps:jessie
 
-RUN apt-get update && apt-get install -y \
+RUN apt-get update && apt-get install -y --no-install-recommends \
 		ca-certificates \
-		curl
+		curl \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
Actually, this whole block can/should be removed since `curl` and `ca-certificates` are included in `buildpack-deps`.  Will update shortly.